### PR TITLE
Minor improvements to trenchmap

### PR DESCRIPTION
**Pull request by vortex1942**
[Game Objects][bug][Code Quality]
## About the PR 

1. Inverts Wall&Ground colors on Oshan maps
2. Fixes pixel gap on the bottom left of maps
3. Uses transparency for Space/Seafloor tiles (Filled in via css)
4. Andd I got rid of a magic number

The pixel gap
image

Tested on both ocean and space maps:
image
image

## Why's this needed? 
1. Something impassable should be darker than a passable area
2. Cuts off the right side of the map, and is annoying
3. Marginal filesize improvement, and brings it inline with goonhub images

### CODE REVIEW

File: code/datums/controllers/sea_hotspot_controls.dm [hunk 1]
```diff
@@ -39,42 +39,47 @@
...
 	#ifdef UNDERWATER_MAP
 	var/list/map_colors = list(
-		empty = rgb(0, 0, 50),
-		solid = rgb(0, 0, 255),
+		empty = rgb(0, 0, 255),
+		solid = rgb(0, 0, 50),
 		station = rgb(255, 153, 58),
-		other = rgb(120, 200, 120))
+		other = rgb(120, 200, 120),
+		transparant = null)
```
Comment (ZeWaka) [inline code/datums/controllers/sea_hotspot_controls.dm:46]:
```suggestion
		transparent = null)
```

File: code/datums/controllers/sea_hotspot_controls.dm [hunk 2]
```diff
@@ -39,42 +39,47 @@
...
 	#else
 	var/list/map_colors = list(
 		empty = rgb(30, 30, 45),
 		solid = rgb(180,180,180),
 		station = rgb(27, 163, 186),
-		other = rgb(186, 0, 60))
+		other = rgb(186, 0, 60),
+		transparant = null)
```
Comment (ZeWaka) [inline code/datums/controllers/sea_hotspot_controls.dm:53]:
```suggestion
		transparent = null)
```

File: code/datums/controllers/sea_hotspot_controls.dm [hunk 3]
```diff
@@ -39,42 +39,47 @@
...
 			var/turf_color = null
+			var/size_mult = 2
 			for (var/x in 1 to world.maxx)
 				for (var/y in 1 to world.maxy)
 					var/turf/T = locate(x,y,MINING_Z)
 					if (istype(T, /turf/simulated/wall/auto/asteroid) || istype(T, /turf/simulated/floor/plating/airless/asteroid))
 						turf_color = "solid"
 					else if (istype(T, /turf/space))
-						turf_color = "empty"
+						turf_color = "transparant"
```
Comment (ZeWaka) [inline code/datums/controllers/sea_hotspot_controls.dm:68]:
```suggestion
						turf_color = "transparent"
```

Comment (ZeWaka) [inline code/datums/controllers/sea_hotspot_controls.dm:68]: Why even bother drawing then? Why not just ignore processing this?

Comment (vortex1942) [reply code/datums/controllers/sea_hotspot_controls.dm]: I definitely overlooked that, I just committed a corrected version that is much simpler

**Review by ZeWaka — APPROVED** [review]

**Pull request merged**